.github/workflows/deploy-familie-tilbake-alerts.yaml

```diff
@@ -0,0 +1,17 @@
+name: Deploy familie-tilbake-alerts
+
+on:
+  workflow_dispatch:
+
+jobs:
+  deploy-prod:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: deploy familie-tilbake-alerts to prod
+        uses: nais/deploy/actions/deploy@v2
+        env:
+          APIKEY: ${{ secrets.NAIS_DEPLOY_APIKEY }}
```

Comment:
Ikke lenger nødvending
https://nav-it.slack.com/archives/C01DE3M9YBV/p1702893023861639